client/components/app/app.js

```diff
@@ -8,113 +8,152 @@ import MediaCarousel from '../mediaCarousel/mediaCarousel';
-    },
-  });
-
-  const [currentlyDisplaying, setCurrentDisplay] = useState('VIDEOS');
-
-  const [featuredMedia, setFeaturedMedia] = useState(dummyData.videos[0]);
+      }
+    }
+  }
 
-  useEffect(() => {
+  componentDidMount() {
     const queryString = window.location.search;
     const params = new URLSearchParams(queryString);
     const id = params.get('id');
     if (id) {
-      $.ajax(`/games/${id}`, {
+      $.ajax(`http://localhost:3002/games/${id}`, {
         method: 'GET',
         success: (results) => {
-          setDummyData(results);
-          setFeaturedMedia(results.videos[0]);
+          this.setState({
+            dummyData: results,
+            featuredMedia: results.videos[0],
+          })
         },
         error: (err) => console.error(err),
       });
     }
-  }, {dummyData});
+  };
 
-  const handleTabButtonClick = (e) => {
+  handleTabButtonClick(e) {
     const text = e.target.innerText;
-    if(currentlyDisplaying === text) {
+    if(this.state.currentlyDisplaying === text) {
       return;
     }
-    setCurrentDisplay(text);
-    text === 'SCREENSHOTS'
-    ?
-    setFeaturedMedia(dummyData.screenshots[0])
-    :
-    setFeaturedMedia(dummyData.videos[0])
+    this.setState({
+      currentlyDisplaying: text,
+    })
+
+    if (text === 'SCREENSHOTS') {
+      this.setState({
+        featuredMedia: this.state.dummyData.screenshots[0],
+      })
+    } else {
+      this.setState({
+        featuredMedia: this.state.dummyData.videos[0],
+      })
+    }
   }
 
-  const handleImageClick = (e) => {
+  handleImageClick(e) {
     const targetId = Number(e.target.attributes.id.value);
-    if (currentlyDisplaying === 'SCREENSHOTS') {
-      for (let i = 0; i < dummyData.screenshots.length; i++) {
-        if (dummyData.screenshots[i].id === targetId) {
-          setFeaturedMedia(dummyData.screenshots[i]);
+    e.target.style
```

Comment:
what's this for?

Reply:
Yeah, agreed. This looks like a remnant of something.

Reply:
are y'all asking what the function is for?

Reply:
`e.target.style` is just hanging there

Reply:
oh lol, thanks guys